### PR TITLE
Add coverage for GET verb and websocket handshake to SCC exec tests.

### PR DESCRIPTION
**Pull request by benluddy**

### CODE REVIEW

**Comment by deads2k** [comment]
Please expand to include websockets.

**Comment by deads2k** [comment]
/approve
/lgtm
/hold in case you want standa to look

release the hold at your discretion.

**Comment by benluddy** [comment]
/cc @stlaz

For context, we were concerned about the possibility that `security.openshift.io/SCCExecRestrictions` might be bypassed for GET pods/exec websocket requests (https://issues.redhat.com/browse/OCPBUGS-4924). There were a few false positive reports of this that turned out to be explainable either by namespace runlevel or by pod specs matching the restricted SCC.

File: test/extended/security/scc.go [hunk 1]
```diff
@@ -54,6 +57,81 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
...
+			conn.Close()
+			g.Fail("non-nil error expected")
+		}
+		o.Expect(err.Error()).To(o.HaveSuffix("bad status"))
+
+		// The underlying golang.org/x/net/websocket API can return a "bad status" error but
+		// does not expose the actual received status code or response body, so the test
+		// continues by demonstrating that authorizing the requester to use the privileged
+		// SCC allows an identical request to successfully complete the websocket
+		// handshake. This is also why a scheduleable pod is created and waited on.
```
[the remark this answers is not in the document]
Comment (benluddy) [reply test/extended/security/scc.go:107]: related: [x/net/websocket: remedy neglect](https://github.com/golang/go/issues/18152)

**Review by dinhxuanvu — COMMENTED** [review]
/lgtm

**Pull request merged**